.github/workflows/python-min-deps.yml

```diff
@@ -0,0 +1,172 @@
+name: Python Dependency Minimum Versions
+
+on:
+  push:
+    branches:
+      - "develop"
+      - "feature/**"
+  pull_request:
+    types: [opened, synchronize, reopened]
+  # Allows workflow to be called from other workflows
+  workflow_call:
+    inputs:
+      ref:
+        required: true
+        type: string
+      force-canary:
+        description: |
+          Forces the current build to be canary.
+          Canary builds test all Python versions and do not use constraints.
+        default: false
+        type: boolean
+      constraints-branch:
+        description: "The name of the branch from which the constraints files will be downloaded or compared with."
+        default: "constraints-develop"
+        type: string
+    secrets:
+      PARAMETER_PASSWORD:
+        description: "Token passed from caller workflows for snowflake integration tests"
+        required: true
+
+# Avoid duplicate workflows on same branch
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-python
```

Comment:
Think this concurrency group name will have to be updated since our `python-versions.yml` uses the same one

Reply:
I think `github.workflow` will be different, since the name and file path for this workflow are different.

Reply:
Ah, ok. That wasn't clear from the GHA docs. I'll update the conc group then.